src/btree/bt_sync.c

```diff
@@ -248,9 +266,56 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
                     btree->rec_max_txn = mod->rec_max_txn;
                 if (mod != NULL && btree->rec_max_timestamp < mod->rec_max_timestamp)
                     btree->rec_max_timestamp = mod->rec_max_timestamp;
+
+                /*
+                 * Check whether the tree is lookaside btree and verify whether the page contents
+                 * are obsolete or not by checking the visibility of the page stop time pair.
+                 */
+
+                if (is_las) {
+                    obsolete = false;
+
+                    __wt_ref_is_obsolete(session, walk, &obsolete);
+
+                    if (obsolete) {
+                        /*
+                         * Get the duplicate tree walk point before marking the current tree page as
+                         * deleted, because the duplicate tree walk expects the ref state should be
+                         * in memory.
+                         */
+                        WT_ERR(__sync_dup_walk(session, walk, flags, &prev));
+
+                        /*
+                         * Mark the page as empty and also set the parent page as dirty. This is to
+                         * ensure when the parent page is checkpointing, the empty child page will
```

Comment:
Maybe replace the word `empty` with `unneeded`.

Reply:
I went with your above suggestion of deleted.